config/reliefweb_import.plugin.importer.wfp_logcluster.yml

```
@@ -0,0 +1,18 @@
+enabled: 1
+max_import_attempts: '3'
+classification:
+  enabled: 1
```

```suggestion
  enabled: false
```

Classification should be disable for the log cluster importer ?

---

Yes